### ISSUE TITLE
The difficult cases in mapping of Penn Treebank POS to UD POS

### ISSUE THREAD

**Issue by manning** [UniversalDependencies/docs#157]
@dan-zeman produced a conversion between Penn Treebank POS tags and UD POS tags using Interset. Separately @sebschu and @manning produced a converter to do the same thing (but with context-sensitive rules operating on PTB trees for the cases where you do not get enough information from PTB tags). I thought it might be useful to work out "inter-annotator agreement".  At the bottom is where our current efforts differ.

Interset is just wrong on RP, @dan-zeman. That is clearly documented: http://universaldependencies.github.io/docs/u/pos/ADP.html and http://universaldependencies.github.io/docs/u/pos/PART.html .

And I think the same is true for `LS`, since that has in practice been used in the PTB WSJ only for alphanumeric list item markers, and you just wrote in the thread on that that "The POS tag is not PUNCT because it is an alphanumeric character."

The remaining choices where we have different translations (`AFX`, `EX`, `PRP$`, and `WP$`) are more interesting. I don't immediately want to say that any of Stanford's choices are wrong – after all, we chose them – but they're not obviously right either, and I can see arguments for going in a different direction. In particular, the choice for `PRP$` and `WP$` touches on the same discussion of `PRON` vs. `DET` that we have been having on other threads. So, it seems like we should try to sort out some principles for that going forward.

––
AFX:
Stanford --> X
Interset --> ADJ

EX:
Stanford --> DET
Interset --> ADV

LS:
Stanford --> X
Interset --> PUNCT

PRP$:
Stanford --> PRON
Interset --> DET

WP$:
Stanford --> PRON
Interset --> DET

RP:
Stanford --> ADP
Interset --> PART

ADD:
Stanford --> X
Interset -->	[missing]

GW:
Stanford --> X
Interset -->	[missing]

NFP:
Stanford --> SYM
Interset -->	[missing]

XX:
Stanford --> X
Interset -->	[missing]

**Comment by sebschu** [comment]
@manning, @ngiordani, @tdozat and I went through this list in our last meeting. We decided to make only one change to our mapping, namely changing EX from DET to PRON for the following two reasons:

1. *it* is an actual pronoun, also in its pleonastic form.
2. The existential *there* functions as an NP and not as a determiner.

I updated our converter and I also updated the documentation. The change will also be reflected in our next data release.
